.github/workflows/terraform.yml

```
@@ -68,6 +231,79 @@
+        with:
+          terraform_version: 1.5.0
+
+      - name: Rollback to Previous State
+        working-directory: terraform
+        run: |
+          terraform state pull > current.tfstate
+          terraform state push previous.tfstate
+          terraform apply -auto-approve
+
+      - name: Notify Rollback Status
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)